configloader/image/tests/test_config.py

```diff
@@ -1,8 +1,8 @@
 import json
 import pathlib
-import hashlib
 
 from dffml.util.asynctestcase import AsyncTestCase
+from dffml.util.net import secure_hash
```

Comment:
`util: crypto` in the issue description means that `*_hash` functions should go in `dffml/util/crypto.py`

Reply:
Fixed that! hash helper functions are now  in dffml/util/crypto.py